css/partials/_locations.scss

```diff
@@ -197,6 +197,57 @@
 	}
 }
 
+.profile-content {
```

Comment:
Can we format these rules with nested selectors, similar to what the rest of this file (and other .scss files) use?